app/src/main/java/com/example/lunark/util/ClientUtils.java

```diff
@@ -48,4 +50,6 @@ public static OkHttpClient test() {
     public static PropertyService propertyService = retrofit.create(PropertyService.class);
     public static ReservationService reservationService = retrofit.create(ReservationService.class);
     public static AccountService accountService = retrofit.create(AccountService.class);
+    public static ReviewReportService reviewReportService = retrofit.create(ReviewReportService.class);
```

Comment:
After all the suggestions are applied, this will be unused. Furthermore, the `retrofit` in `ClientUtils` does not set the Authorization header, so using this will likely break something in the future.
```suggestion
```